converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/ImagingStudy/ImagingStudyInstancePropertySynchronizer.cs

```diff
@@ -26,14 +44,54 @@ public void Synchronize(FhirTransactionContext context, ImagingStudy.InstanceCom
                 return;
             }
 
-            // Add sopclass to instance
+            SynchronizePropertiesAsync(instance, context, true, AddSopClass, cancellationToken);
```

Comment:
can the relationships between instance, synchronizeAction, and required be expressed more directly as a property or table mapping. and then added to each class?

Reply:
This is a good idea, I made it so that it is mapped in a list and then we just loop through that instead